src/boot_fmt/core.clj

```diff
@@ -23,6 +23,15 @@
         (clojure.string/split #"\000")
         set)))
 
+(defn find-files-git-cached
+  []
+  (let [{:keys [exit out err]} (clojure.java.shell/sh "git" "diff"
+                                                      "--cached" "--name-only")]
+    (when (not= exit 0) (throw (ex-info "git diff --cached failed" {:err err})))
+    (->> (clojure.string/split out #"\n")
+         (filter #(re-matches #".*\.(clj|cljs|cljc|cljx|cljs\.hl|boot)$" %))
```

Comment:
why not use `git diff --cached --name-only "*.clj" "*.cljs" ...`? This would make this very similar to `find-files-git`

Reply:
also the duplication concerns me, you need to add new file extensions in two places